test/functional/p2p_permissions.py

```
@@ -112,7 +112,7 @@ def in_mempool():
             self.bump_mocktime(1)
             return txid in self.nodes[0].getrawmempool()
 
-        with self.nodes[1].assert_debug_log(["Force relaying tx {} from peer=0".format(txid)]):
+        with self.nodes[1].assert_debug_log(["Force relaying tx {} (wtxid={}) from peer=0".format(txid, tx.getwtxid())]):
```

Ruff (0.12.2)

Use f-string instead of `format` call

Convert to f-string

(UP032)